src/main/java/uk/gov/companieshouse/mapper/ReasonMapper.java

```diff
@@ -9,24 +9,26 @@
 public class ReasonMapper implements Mapper<ReasonEntity, Reason> {
 
     private final OtherReasonMapper otherReasonMapper;
+    private final IllnessReasonMapper illnessReasonMapper;
 
-    public ReasonMapper(OtherReasonMapper otherReasonMapper) {
+    public ReasonMapper(OtherReasonMapper otherReasonMapper, IllnessReasonMapper illnessReasonMapper) {
         this.otherReasonMapper = otherReasonMapper;
+        this.illnessReasonMapper = illnessReasonMapper;
```

Comment:
Can this be refactored please? Using constructor parameters will be difficult to read as more reasons are added. Will autowiring work for these mapper dependencies?

Reply:
Resolved on https://github.com/companieshouse/lfp-appeals-api/pull/112